src/dsp/delay.cpp

```diff
@@ -5,7 +5,7 @@
 #include "delay.h"
 #include <string.h>
 
-#define USE_AVX 0
+#define USE_AVX 1
```

Comment:
Detta känns dåligt på många sätt

https://stackoverflow.com/questions/28939652/how-to-detect-sse-sse2-avx-avx2-avx-512-avx-128-fma-kcvi-availability-at-compile

kanske något sånt istället?

Reply:
Ja och nej, vi tappar viss precision i och med att kompilatorn gör sina optimeringar på sitt sätt, det är inte alltid det blir AVX utan detta kräver att det blir det.

Reply:
[c63a8bb](https://github.com/acoustic-warfare/beamforming-lk/pull/103/commits/c63a8bbf0b91554cef1e05858c175e698b4fb871)